jetstream/tests/test_analysis.py

```diff
@@ -380,7 +382,16 @@ def test_create_subset_metric_table_query_univariate_basic(experiments):
     assert expected_query == actual_query
 
 
-def test_create_subset_metric_table_query_covariate_basic(experiments, monkeypatch):
+@pytest.mark.parametrize(
+    ("randomization_unit"),
+    [
+        RandomizationUnit.GROUP_ID,
+        RandomizationUnit.NIMBUS,
+        RandomizationUnit.NORMANDY,
+        RandomizationUnit.USER_ID,
```

Comment:
Are these all the possible values for RandomizationUnit provided by Experimenter?

Reply:
Yes, and actually I could probably just make that more explicit by doing something like:
```py
[r for r in RandomizationUnit]
```